### PR TITLE
Update the GraphQL help text with schema generation command

### PR DESCRIPTION
**Pull request by DimuthuMadushan**
Fixes: [#3737](https://github.com/ballerina-platform/ballerina-standard-library/issues/3737)

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help [hunk 1]
```diff
@@ -1,27 +1,40 @@
 NAME
-       ballerina-graphql - Generate Ballerina client sources from a GraphQL config file 
-                        configured with GraphQL schemas(SDL) and GraphQL queries.
+       ballerina-graphql - Generate Ballerina client sources for a GraphQL config file
+       and generate GraphQL schema for a Ballerina GraphQL service.
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help:3]:
```suggestion
       and generate the GraphQL schema for a Ballerina GraphQL service.
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help [hunk 2]
```diff
@@ -1,27 +1,40 @@
 NAME
-       ballerina-graphql - Generate Ballerina client sources from a GraphQL config file 
-                        configured with GraphQL schemas(SDL) and GraphQL queries.
+       ballerina-graphql - Generate Ballerina client sources for a GraphQL config file
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help:2]:
```suggestion
       ballerina-graphql - Generate the Ballerina client sources for a GraphQL config file
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help [hunk 3]
```diff
@@ -1,27 +1,40 @@
...
 OPTIONS
-        -i, --input <path>
-           Path to a GraphQL config file configured with GraphQL schemas(SDL) and GraphQL queries.
-
-        -o, --output <path>
-           Location of the generated Ballerina source code. If this path is not specified,
-           the output will be written to the same directory from which the command is run.
-
+        -i, --input <graphql-configuration-file-path | ballerina-graphql-service-file-path>
+            This is mandatory input. The given GraphQL config file input configured with GraphQL
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help:25]:
```suggestion
            This is mandatory input. The given GraphQL config file input is configured with GraphQL
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help [hunk 4]
```diff
@@ -1,27 +1,40 @@
...
+            schemas(SDL) and queries will generate the Ballerina GraphQL client sources. The given
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help:26]:
```suggestion
            schemas (SDL) and the queries will generate the Ballerina GraphQL client sources. The given
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help [hunk 5]
```diff
@@ -1,27 +1,40 @@
...
+            Ballerina GraphQL service file input will generate the GraphQL schema(SDL) file
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help:27]:
```suggestion
            Ballerina GraphQL service file input will generate the GraphQL schema (SDL) file
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help [hunk 6]
```diff
@@ -1,27 +1,40 @@
...
+            relevant to the service.
+        -o, --output <output-location>
+            Location of the generated Ballerina source code or GraphQL schema. If this path is not
+            specified, the output will be written to the same directory from which the command is
+            run.
+        -s, --service <service-base-path>
+            This service base path is used to identify the service that needs to generate the
+            GraphQL schema. This option is used with the command of GraphQL schema generation.
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help:35]:
```suggestion
            GraphQL schema. This option is used with the GraphQL schema generation command.
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help [hunk 7]
```diff
@@ -40,8 +40,8 @@ COMMANDS
         format          Format Ballerina source files
         grpc            Generate the Ballerina sources for a given Protocol
                         Buffer definition
-        graphql         Generate Ballerina client sources
-                        for a given GraphQL schema(SDL) and GraphQL queries
+        graphql         Generate Ballerina client sources for a GraphQL config file
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help:43]:
```suggestion
        graphql         Generate the Ballerina client sources for a GraphQL config file
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help [hunk 8]
```diff
@@ -40,8 +40,8 @@ COMMANDS
...
+                        and generate GraphQL schema for a Ballerina GraphQL service.
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help:44]:
```suggestion
                        and generate the GraphQL schema for a Ballerina GraphQL service.
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help [hunk 9]
```diff
@@ -1,27 +1,40 @@
...
+                   [-o | --output] <output-location>
+                   [-s | --service] <service-base-path>
 
 
 DESCRIPTION
-       Generate the Ballerina GraphQL client sources for a given GraphQL config file 
-       configured with GraphQL schemas(SDL) and GraphQL queries.
+       Generate the Ballerina GraphQL client sources for a given GraphQL config file configured
+       with GraphQL schemas specified by GraphQL Schema Definition Language(SDL) and GraphQL
+       queries or export a GraphQL schema(SDL) for a given Ballerina GraphQL service.
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help:17]:
```suggestion
       queries or export a GraphQL schema (SDL) for a given Ballerina GraphQL service.
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help (hunk 3 above)
Comment (DimuthuMadushan) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help]: This doesn't convey the exact message. Hence changed as follows. `The given GraphQL config file which is configured with GraphQL schemas (SDL) and queries, will generate the Ballerina GraphQL client sources.`

**Review by praneesha — APPROVED** [review]

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**